bot/commands/utility/listReminders.js

```diff
@@ -14,8 +14,10 @@ function listReminders(message, args, config) {
         .then((jobs) => {
             const list = jobs.filter((j) => !j.attrs.repeatInterval)
             // eslint-disable-next-line no-underscore-dangle
```

Comment:
Disable this rule for the file because theres multiple places its being used, i think its something like:
```
/* eslint-disable no-underscore-dangle */
```